python/lsst/obs/lsstSim/lsstSimMapper.py

```diff
@@ -381,6 +381,20 @@ def std_eimage(self, item, dataId):
         """Standardize a eimage dataset by converting it to an Exposure instead of an Image"""
         return self._standardizeExposure(self.exposures['eimage'], item, dataId, trimmed=True)
 
+    def _createInitialSkyWcs(self, exposure):
+        """Create a SkyWcs from the header metadata.
+
+        PhoSim data may not have self-consistent boresight and crval/crpix
+        values, and/or may have been written in FK5, so we just use the
+        metadata here, and ignore VisitInfo/CameraGeom.
+
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.Exposure`
+            The exposure to get data from, and attach the SkyWcs to.
+        """
+        self._createSkyWcsFromMetadata(exposure)
```

Comment:
This is essentially the same as the `_createInitialSkyWcs` in obs_base; could that just be imported and used here?

Reply:
Note the docstring of this method. We *only* use the metadata SkyWcs for lsstSim, because PhoSim would write nonsense for the things that the smarter SkyWcs code needed.